mathesar/tests/conftest.py

```diff
@@ -184,3 +184,12 @@ def _create_column(table, column_data):
         column = mathesar_model_column.current_objects.get_or_create(attnum=attnum, table=table)
         return column[0]
     return _create_column
+
+
+@pytest.fixture
+def custom_types_schema(test_db_model, schema, live_server):
```

Comment:
The fixture returns a URL and not a schema, please change the fixture name to something like `custom_types_schema_url`